src/Microsoft.VisualStudio.Services.Agent/JobNotification.cs

```diff
@@ -55,7 +80,54 @@ public sealed class JobNotification : AgentService, IJobNotification
                 await _outClient.ConnectAsync(cancellationToken);
                 _writeStream = new StreamWriter(_outClient, Encoding.UTF8);
                 _configured = true;
-                Trace.Info("Connection successfull to named pipe {0}", pipeName);
+                Trace.Info("Connection successful to named pipe {0}", pipeName);
+            }
+        }
+
+        public void StartClient(String socketAddress)
```

Comment:
should we throw on StartClient not succeed?
What will happen, if the agent fail to start the notification channel but still still run the job?